Assets/MRTK.ThirdParty/MRTK-Quest/Scripts/Config/MRTKOculusConfig.cs

```diff
@@ -0,0 +1,297 @@
+//------------------------------------------------------------------------------ -
+//MRTK - Quest
+//https ://github.com/provencher/MRTK-Quest
+//------------------------------------------------------------------------------ -
+//
+//MIT License
+//
+//Copyright(c) 2020 Eric Provencher
+//
+//Permission is hereby granted, free of charge, to any person obtaining a copy
+//of this software and associated documentation files(the "Software"), to deal
+//in the Software without restriction, including without limitation the rights
+//to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//copies of the Software, and to permit persons to whom the Software is
+//furnished to do so, subject to the following conditions :
+//
+//The above copyright notice and this permission notice shall be included in all
+//copies or substantial portions of the Software.
+//
+//THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+//AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+//SOFTWARE.
+//------------------------------------------------------------------------------ -
+
+using UnityEngine;
+using UnityEngine.XR;
+
+namespace prvncher.MixedReality.Toolkit.Config
+{
+#if UNITY_EDITOR
+    [UnityEditor.InitializeOnLoad]
+#endif
+    [CreateAssetMenu(menuName = "MRTK-Quest/MRTK-OculusConfig")]
+    public class MRTKOculusConfig : ScriptableObject
+    {
+        /// <summary>
+        /// Enum used for controlling the teleport pointer activated by MRTK Quest controllers.
+        /// </summary>
+        public enum TeleportPointerMode
+        {
+            Custom,
```

Comment:
Nit: Would be awesome to have some comments on these - i.e. what is custom, what is official?